dawn/examples/python/data/unstructured_stencil_ref.cpp

```diff
@@ -0,0 +1,44 @@
+namespace dawn_generated{
+namespace cxxnaiveico{
+
+class unstructured_stencil {
+private:
+
+  struct stencil_13 {
+    Mesh const& m_mesh;
+    Field<double>& m_in;
```

Comment:
there is a change coming in this API very soon - fields need to be specialized. Do you think it is worth it to have a reference here (even though it might be changing quite frequently in the coming weeks)

Reply:
I agree. Removed the verification for this example for the moment.